README.md

```diff
@@ -137,6 +138,24 @@ The parameters are as follows:
 	#(box.with(stroke: (left: 1pt + black), inset: (left: 5pt, y: 6pt)))[which displays like this.]
 	-->
 
+- `mitex`:
+  A `mitex` callback function to be used when math equation is encountered in the Markdown,
+  or `none` if math equation should be treated as normal text.
+  Because Typst does not support latex math equation natively,
+  the user must configure this.
+  - Accepted values: The `mitex` function imported from [mitex](https://typst.app/universe/package/mitex) package, or `none`.
```

Comment:
I think we should generalize this: Any function of the form `f(block, content)` works, right?